test/components/chromosome.js

```diff
@@ -5,13 +5,13 @@ describe("<ChromosomeView />", function(){
         chromosome = drake.getGenotype().chromosomes[1]["a"];
 
   it("should create a <div> tag with appropriate classes", function() {
-    const wrapper = shallow(<ChromosomeView org={drake} chromosomeName='1' side='a'/>);
+    const wrapper = shallow(<ChromosomeView chromosome={chromosome}/>);
```

Comment:
This test was the only place I could find `ChromosomeView` taking in a `side` and `chromosomeName` instead of a `chromosome`. Since it always receives a `chromosome`, and this information is redundant in that case, I removed the alternate "constructor" for `ChromosomeView`